CervAnn/synonysaurus
# Setup search

#### What's this PR do?
This PR initializes the SearchForm functionality and renders the synonyms to the DOM
#### Where should the reviewer start?
The reviewer should start in App.vue, then SearchForm.vue, and then Result.vue
#### How should this be manually tested?
This should be manually tested by providing an input term to search, then pressing the submit button.
#### Any background context you want to provide?
Learning to pass props has been a steep learning curve; I'm continuing to work through passing the search functionality to each result via the button rendered for each result component
#### What are the relevant tickets?
Issue #1, Issue #2, Issue #3, Issue #4, Issue #5, Issue #6, and Issue #7
#### Screenshots (if appropriate)

<img width="1439" alt="Screen Shot 2019-11-05 at 2 08 59 PM" src="https://user-images.githubusercontent.com/11339301/68246378-e664e680-ffd5-11e9-8edf-45bc029d62d5.png">

<img width="1439" alt="Screen Shot 2019-11-05 at 2 09 10 PM" src="https://user-images.githubusercontent.com/11339301/68246387-e95fd700-ffd5-11e9-96e0-8d0245fcf181.png">
